app/src/renderer/components/network/PageNetwork.vue

```diff
@@ -69,10 +69,10 @@
         <tbody>
           <tr v-if="blocks.length === 0" class="block data-table__row">
             <td>
-              <img class="loading-icon" src="~assets/images/loader.svg" />
```

Comment:
@sabau flagging this for you as i believe we have some linting conflict about this rule

Reply:
Ok, I'll try to fix in the lint lint lint PR :) which rules are conflicting?

Reply:
probably how we handle self closing elements (with a slash or without, with a space or without)